server/src/shared/app/base-controller.ts

```diff
@@ -16,8 +16,8 @@ export abstract class BaseController {
   }
 
   public redirect<Res extends express.Response>(res: Res, url: string, params: ParamList) {
-    res.status(200).redirect(params.getFormattedUrlWithParams(url))
-    return this.ok(res)
+    res.redirect(params.getFormattedUrlWithParams(url))
+    return res
```

Comment:
I'm curious what the change here does; what is the difference between the two implementations?

Reply:
It was overwriting the 302 redirect status of the res. This status code is what instructs the frontend browser to change its page target.